### ISSUE TITLE
RFC: Move Category Block and Include Images

### ISSUE THREAD

**Issue by mdroidian** [Smile-SA/elasticsuite#1056]
Also, help wanted.

What is the best approach to move categories out of (before) layered navigation?

Many sites do this, especially for larger category counts.

![homedepot](https://user-images.githubusercontent.com/3792666/44938515-bae4cd00-ad3c-11e8-89d6-126d49a9663b.JPG)
![bestbuy](https://user-images.githubusercontent.com/3792666/44938516-bae4cd00-ad3c-11e8-8899-0b38a43319f1.JPG)
![amazon](https://user-images.githubusercontent.com/3792666/44938517-bae4cd00-ad3c-11e8-8cfa-d0aa499b5e06.JPG)

**Comment by romainruaud** [comment]
Hello,

it's not something really related with Elasticsuite... You could do it easily with a custom category template : 

- your template loop on current category childrens
- you have another image attribute on your categories (like "merchandising image").
- during the loop, you display it

I don't know how we can provide much support on this one.

Regards

**Comment by mdroidian** [comment]
Yes, we had this already.  The problem with that would be querying the category children and category children count again when it is already being queried by Elasticsuite.  This results in unnecessary overhead and page load time.
